lib/cdo/shared_constants.rb

```diff
@@ -0,0 +1,16 @@
+# A set of constants shared between ruby and JS via /shared/config/sharedConstants.json
+require 'json'
+
+module SharedConstants
+  @@json = nil
+
+  def self.json
+    return @@json if @@json
+
+    file = File.read(File.join(__dir__, '../../shared/config/constants.json'))
+    # @@json = JSON.parse(file)
+    @@json = JSON.parse(file, object_class: OpenStruct)
```

Comment:
I suspect that rather than doing this at runtime on first access, we might prefer to have some build step that generates a static rb file from our json file? Do we have any sort of precedence for something like that, or is this okay?